docusaurus.config.js

```diff
@@ -47,10 +47,17 @@ module.exports = {
         sidebarPath: require.resolve("./optimize_sidebars.js"),
         editUrl: "https://github.com/camunda/camunda-docs/edit/main/",
         versions: {
+          "3.11.0": {
+            label: "8.3 / 3.11.0",
+          },
```

Comment:
This label has most recently been discussed [in Slack](https://camunda.slack.com/archives/C55U06YRG/p1696455410241709?thread_ts=1696454303.091489&cid=C55U06YRG). 

Note that this label only applies to Optimize documentation.

This is what it looks like when viewing an Optimize page: 

<img width="780" alt="image" src="https://github.com/camunda/camunda-docs/assets/1627089/3371181c-9e19-43c3-8fa7-0d462f9a91b3">